hyperon_das_atomdb/adapters/redis_mongo_db.py

```diff
@@ -703,45 +679,59 @@ def get_matched_links(
         if link_type != WILDCARD and WILDCARD not in target_handles:
             try:
                 link_handle = self.get_link_handle(link_type, target_handles)
-                return None, [link_handle]
+                return [link_handle]
             except AtomDoesNotExist:
-                return None, []
+                return []
 
-        link_type_hash = WILDCARD if link_type == WILDCARD else self._get_atom_type_hash(link_type)
+        link_type_hash = (
+            WILDCARD if link_type == WILDCARD else ExpressionHasher.named_type_hash(link_type)
+        )
 
         # NOTE unreachable
         if link_type in UNORDERED_LINK_TYPES:
             target_handles = sorted(target_handles)
```

Comment:
ditto.

Reply:
Not fixed. I just created a card for this: https://github.com/singnet/das-atom-db/issues/218